docs/standard/data/xml/xslt-transformations.md

```diff
@@ -18,9 +18,9 @@ ms.workload:
   - "dotnetcore"
 ---
 # XSLT Transformations
-The Extensible Stylesheet Language Transformation (XSLT) lets you transform the content of a source XML document into another document that is different in format or structure. For example, you can use XSLT to transform XML into HTML for use on a Web site or to transform it into a document that contains only the fields required by an application. This transformation process is specified by the [W3C XSL Transformations (XSLT) Version 1.0 recommendation](http://go.microsoft.com/fwlink/?LinkID=49919).  
+The Extensible Stylesheet Language Transformation (XSLT) lets you transform the content of a source XML document into another document that is different in format or structure. For example, you can use XSLT to transform XML into HTML for use on a Web site or to transform it into a document that contains only the fields required by an application. This transformation process is specified by the [W3C XSL Transformations (XSLT) recommendation](https://www.w3.org/TR/xslt/).  
```

Comment:
Given @pjanotti prompt answer, I'll have to take it back my previous feedback on this. So let's switch back to https://www.w3.org/TR/xslt-10/ and add the Version 1.0 back here.

Let me know if you'd like me to do that @pkulikov given I asked you to change that before.

Reply:
@mairaw no problem: I'll change it back! Please create the issue for un-obsoleting another type and that PR can be merged at last :).